lib/logging/logging.dart

```diff
@@ -578,12 +608,57 @@ class LogDetailsUI extends CoreElement {
   CoreElement content;
   CoreElement message;
 
+  InspectorTreeHtml tree;
+
   void setData(LogData data) {
     // Reset the vertical scroll value if any.
     content.element.scrollTop = 0;
 
     this.data = data;
 
+    tree = null;
+    if (data.node != null) {
+      message.clear();
+      tree = InspectorTreeHtml(
+        summaryTree: false,
+        treeType: FlutterTreeType.widget,
+        onSelectionChange: () {
+          final InspectorTreeNode node = tree.selection;
+          if (node != null) {
+            tree.maybePopulateChildren(node);
+          }
+          // The ui wasn't updated because this selection was triggered by the logging
+          // view instead of the regular inspector.
+          node.diagnostic.setSelectionInspector(false);
+          // TODO(jacobr): warn if the selection can't be set as the node is
+          // stale which is likely if this is an old log entry.
+        },
+        onExpand: (e) {
+          print('XXX expand $e');
+        },
+        onHover: (e) {
+
+        },
+      );
+
+      final InspectorTreeNode root = tree.setupInspectorTreeNode(
+        tree.createNode(),
+        data.node,
+        expandChildren: true,
+        expandProperties: true,
+      );
+      // No sense in collapsing the root node.
+      root.allowExpandCollapse = false;
+      tree.root = root;
+      message.add(tree.element);
+
+      if (_verboseDebugging) {
```

Comment:
Delete this block. Adding this div slowed down things too much.

Reply:
👍